lib/winapi/badptr.c

```diff
@@ -0,0 +1,9 @@
+#include <windows.h>
+
+BOOL IsBadWritePtr(LPVOID lp, UINT_PTR ucb)
+{
+  //FIXME: Run NtQueryVirtualMemory for the pages?
+
+  /* We disallow all access for now, because memory is potentially unsafe */
+  return TRUE;
```

Comment:
nit: The rest of winapi uses 4 space indentation.

Reply:
Yes, stupid mistake. Should have been fixed.